encoding/protobuf/error.go

```diff
@@ -98,7 +98,7 @@ func WithErrorDetails(details ...proto.Message) ErrorOption {
 }
 
 // convertToYARPCError is to be used for handling errors on the inbound side.
-func convertToYARPCError(encoding transport.Encoding, err error) error {
+func convertToYARPCError(encoding transport.Encoding, codec *codec, err error) error {
```

Comment:
nit: It looks like we add `codec` as the last argument of the function everywhere, can we do the same here?

Reply:
I opted to make it second because the encoding and codec are closely related, and the error is the important part of this function, but sounds good for consistency 👍